### ISSUE TITLE
[Bug] Menu click tone when first starting the game

### ISSUE THREAD

**Issue by ProPeach** [tsunamayo/Starship-EVO#3323]
### Short Description of the issue:
When launching the game, a tone plays when the game reaches the main menu. It sounds similar to the SFX for a button 

### Steps to reproduce the issue:
Launch the game, listed for the tone when the main menu appears

### Environment
- Starship EVO version: 20w48d
- Graphics Card: rx580

**Comment by Kaiser-Indrasil** [comment]
Yeah, I also noticed it's not affected by sounds settings, so if you've got them set at like 5% and the overall system sounds like I do, it's SUPER LOUD.
